### DISCUSSION TITLE
Allow lximage-qt to drag&drop images from viewing area

### DISCUSSION THREAD

**Discussion by ssrublev** [lxqt/lxqt#2191] [Ideas]
Hello! I've noticed that lximage-qt is unable to allow copying of image by drag-and-dropping it out of the window. This is essential thing that may be used in very basic operations. 

I think that LXQt may compare itself to MATE in terms of very basic functions.

MATE's image viewer allows drag and drop.

E. g. MATE's file manager caja allows creating symlinks. I think pcmanfm-qt should allow it too.

**Comment by stefonarch** [comment]
> unable to allow copying of image by drag-and-dropping it out of the window

Not from the window true, but from the thumbnails bar this is possible.




> caja allows creating symlinks. I think pcmanfm-qt should allow it too.

It can, since ages, DND and...

![schermata-02-14-12-09](https://user-images.githubusercontent.com/10681413/153853384-93ee308a-9a6c-4e32-90f3-f8f0a5395fd0.png)

**Comment by tsujan** [comment]
DND from the view is a bad idea. It's in conflict with scrolling the image (when its size is bigger than that of the view) by grabbing it.

> I think that LXQt may compare itself to MATE in terms of very basic functions.

LXQt may implement good features of other DEs but it doesn't copy other DEs, not even LXDE.
